### DISCUSSION TITLE
v0.11.8

### DISCUSSION THREAD

**Discussion by AsamK** [AsamK/signal-cli#1220] [Releases]
## What's Changed
* Document accepting group invitations by @exquo in https://github.com/AsamK/signal-cli/pull/1218

**Full Changelog**: https://github.com/AsamK/signal-cli/compare/v0.11.7...v0.11.8

<hr /><em>This discussion was created from the release <a href='https://github.com/AsamK/signal-cli/releases/tag/v0.11.8'>v0.11.8</a>.</em>

**Comment by orazioedoardo** [comment]
Invalid PGP signature?

**Reply by AsamK** [reply]
Thanks, should be fixed now.
